### PR TITLE
[#1166] fix(docs): Fix typos `graviton` and syntax problems in docs

### LINKED ISSUE #1166: [Improvement] Spelling errors about Gravition
### What would you like to be improved?

There are still places to use old name - Graviton in document like attached screenshot
截屏2023-12-14 11 49 15

### How should we improve?

replace graviton with gravitino

### PR DESCRIPTION
**Pull request by yuqi1129**
### What changes were proposed in this pull request?

- Fix typo `Graviton` in `index.md` 
- Fix link mistakes in `index.md` and other sentence problems

### Why are the changes needed?

To Improve the professionalism and rigor of documents 

Fixed #1166 

### Does this PR introduce _any_ user-facing change?

N/A.

### How was this patch tested?

N/A.

### CODE REVIEW

File: docs/how-to-test.md [hunk 1]
```diff
@@ -31,15 +31,15 @@ To run the unit test, you can simply run the following command:
 ./gradlew test -PskipITs
 ```
 
-This command runs all the unit tests and skip the integration tests.
+This command runs all the unit tests and skips the integration tests.
 
 ## Run the integration tests
 
-Gravitino has two modes to run the integration tests, the `embedded` and `deploy` modes.
+Gravitino has two modes to run the integration tests, the `embedded`(default) and `deploy` modes.
```
Comment (justinmclean) [inline docs/how-to-test.md:38]: Need space between embedded and "(default)", but I would change to ".. the default embedded mode and the deploy mode."

File: docs/how-to-test.md [hunk 2]
```diff
@@ -31,15 +31,15 @@ To run the unit test, you can simply run the following command:
...
 
 * With the `embedded` mode, the integration test starts an embedded `MiniGravitino` server
   within the same process of the integration test to run the integration tests.
-* With the `deploy` mode, the user has to deploy a Gravitino binary package beforehand, the
+* With the `deploy` mode, the user has to build(`./gradlew compileDistribution`) a Gravitino binary package beforehand, the
```
Comment (justinmclean) [inline docs/how-to-test.md:42]: needs space

File: docs/how-to-test.md [hunk 3]
```diff
@@ -153,7 +153,7 @@ server code, follow these steps:
 
 If a test fails, you can retrieve valuable information from the logs and test report. Test reports are in the `./build/reports` directory. The integration test logs are in the `./integrate-test/build` directory. In deploy mode, Gravitino server logs are in the `./distribution/package/logs/` directory. In the event of a test failure within the GitHub workflow, the system generates archived logs and test reports. To obtain the archive, follow these steps:
 
-1. Click the `detail` link associated with the failed integrate test in the pull request. This redirects you to the job page.
+1. Click the `detail` link associated with the failed integrated test in the pull request. This redirects you to the job page.
```
Comment (justinmclean) [inline docs/how-to-test.md:156]: integrated - > integration

**Comment by jerryshao** [comment]
@yuqi1129 can you please fix the comment addressed by @justinmclean

**Comment by yuqi1129** [comment]
> @yuqi1129 can you please fix the comment addressed by @justinmclean

I see.

File: docs/how-to-test.md (hunk 1 above)
Comment (yuqi1129) [reply docs/how-to-test.md:38]: OK

File: docs/index.md [hunk 4]
```diff
@@ -15,11 +15,11 @@ metadata access for data and AI assets.
...
 
-You can get Graviton from the [GitHub release page](https://github.com/datastrato/gravitino/releases),
-or you can build Gravitino from source, please see [How to build Gravitino](./how-to-build.md).
+You can get Gravitino from the [GitHub release page](https://github.com/datastrato/gravitino/releases),
+or you can build Gravitino from source code, please see [How to build Gravitino](./how-to-build.md).
 
-Gravitino runs on both Linux and macOS, and requires Java 8. Gravitino trino-connector runs with
-Trino, and requires Java 17. This should include JVMs on x86_64 and
+Gravitino runs on both Linux and macOS, and requires at least Java 8. Gravitino trino-connector runs with
+Trino, and requires at least Java 17. This should include JVMs on x86_64 and
```
Comment (yuqi1129) [inline docs/index.md:22]: @jerryshao 
Please take a look at this point.  I think we need to distinguish between the compilation environment and the running environment.

Comment (jerryshao) [reply docs/index.md:22]: Here, it means that trino requires Java 17 runtime, so our trino-connector is compiled against JDK17.

Comment (jerryshao) [reply docs/index.md:22]: Also, Gravitino currently can only run on Java 8, not on Java 11 or 17, so your change of "at least" is not correct.

Comment (jerryshao) [reply docs/index.md:22]: Besides, here it mentions all about the running environment, not the building environment.

Comment (yuqi1129) [reply docs/index.md:22]: So the word `run` means `compilation` and `run`?

As far as I see, we can run the Gravitino server with JDK 17.

Comment (yuqi1129) [reply docs/index.md:22]: I mean we need JDK 8 to 17 to build the Gravitino server and at least JRE 8 to run Gravitino server and require at least JRE 17 to run the Trino connector.

Comment (jerryshao) [reply docs/index.md:22]: It just means run, and it cannot run with JDK17 currently because of JAXB problem. You can try it.

Comment (yuqi1129) [reply docs/index.md:22]: It works well for me after https://github.com/datastrato/gravitino/pull/868

Comment (jerryshao) [reply docs/index.md:22]: I think we don't have a thorough test, we should not guarantee this and modify this document to say that.

Comment (jerryshao) [reply docs/index.md:22]: My previous test has many issues related to jaxb.

Comment (yuqi1129) [reply docs/index.md:22]: Okay, I will revert changes here.

**Review by jerryshao — APPROVED** [review]
LGTM.

**Pull request merged**